packages/fields/src/types.ts

```diff
@@ -39,6 +47,8 @@ export interface BasePost extends CommonPost {
 	permalink_template?: string;
 	date?: string;
 	modified?: string;
+	author?: number;
+	_embedded?: Embedded;
```

Comment:
I'm not sure about this personally. I think `_embedded` is not always present in `BasePost` unless you actually call the REST API with embed=true or something. 

So my first question:  can we do without this in this field? If we can, I think it's better, if we can't, maybe we should probably create a new type `BasePostWithEmbeds` or something that extends `BasePost` and use it, in that new field.

Reply:
I've pushed https://github.com/WordPress/gutenberg/pull/66939/commits/23c43f73090db805be83eb144e77fbfa735b2b9b

I've seen some other issues with the data fetching for this field that should be improved (see conversation below), but I believe it should be looked at separately.